### DISCUSSION TITLE
[4.1] Description of class parameters for images (Intro and Full)

### DISCUSSION THREAD

**Discussion by Kostelano** [joomla/joomla-cms#47085] [2. Cold Features]
### Steps to reproduce the issue
Articles: Options --> Layout Editing, at the bottom there are class options for images.

![Screenshot_1](https://user-images.githubusercontent.com/8440661/163569798-ca29c422-5eae-43bf-99da-bc285acb2935.png)

Install blog demo data. Go to the "Your Template" article, click EDIT and go to the Images and Links tab.
Pay attention to the image classes and the description of the parameter.

![Screenshot_2](https://user-images.githubusercontent.com/8440661/163570182-07576008-3990-4753-a367-b953d186e5af.png)

**REMARKS**

1. The description is not in the parameters of the article component, but is here on the tab and only for the full image. Should I add it for the intro image as well?
2. The default class is **left**, however the **float-start** and **float-end** classes are better styled. See the difference in the frontend. Wouldn't it be better to make **float-start** the default class? Among other things, we will also get a "match" of the description to the real class in the parameter (say, **left** is not mentioned here).

![Screenshot_3](https://user-images.githubusercontent.com/8440661/163571004-1f08e7b3-b676-4f44-9416-258eb2fd23d5.png)

**Comment by brianteeman** [comment]
1.  yes please

**Comment by Kostelano** [comment]
For the first part at Brian's request - https://github.com/joomla/joomla-cms/pull/37563.

On the second part, I would like to receive some comment. Unless the maintainers feel it necessary to set the class "float-start" as the default, please close this.

**Comment by chmst** [comment]
We cannot change this now, it would be a b/c break. Styling the "left" and "right" (with regard on rtl languages) is better.
